package.json

```diff
@@ -1,24 +1,30 @@
 {
-  "name": "@maxkorp/passport-openidconnect",
-  "version": "0.1.0",
+  "name": "@elm-street-technology/passport-openidconnect",
+  "version": "0.1.1",
   "description": "OpenID Connect authentication strategy for Passport.",
-  "keywords": ["passport", "openid", "openidconnect", "auth", "authn", "authentication", "identity"],
+  "keywords": [
+    "passport",
+    "openid",
+    "openidconnect",
+    "auth",
+    "authn",
+    "authentication",
+    "identity"
+  ],
   "repository": {
     "type": "git",
-    "url": "git://github.com/maxkorp/passport-openidconnect.git"
+    "url": "https://github.com/elm-street-technology/passport-openidconnect"
   },
   "bugs": {
     "url": "http://github.com/jaredhanson/passport-openidconnect/issues"
   },
-  "author": {
-    "name": "Jared Hanson",
-    "email": "jaredhanson@gmail.com",
-    "url": "http://www.jaredhanson.net/"
-  },
-  "licenses": [ {
-    "type": "MIT",
-    "url": "http://www.opensource.org/licenses/MIT"
-  } ],
+  "author": "Jared Hanson <jaredhanson@gmail.com> (http://www.jaredhanson.net/)",
+  "licenses": [
```

Comment:
Using an array for licenses is the old way, a string "MIT" should suffice.

https://docs.npmjs.com/files/package.json#license